Update domains.json

sandwedg opened the pull request:
#### Hi! What is your PR about?

- [ ] Add or edit a skin
- [X] Add custom domain for your space
- [ ] Add alias for migrate your space
- ~Add images for your space~ (deprecated) Add the image URL on the space settings directly
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

gokaiorg reviewed (commented):
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/dfrag.eth

sandwedg commented:
> I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/dfrag.eth

Fixed now
